docusaurus/docs/tutorials/ui-extensions/exposed-components.md

```diff
@@ -0,0 +1,77 @@
+---
+id: exposed-components
+title: Exposed components
+description: Share functionality between plugins with exposed components
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - extensions
+  - ui-extensions
+sidebar_position: 30
+---
+
+Exposed components are a way to share functionality between plugins.
```

Comment:
```suggestion
Exposing components makes it possible to easily share functionality between different plugins. Once a plugin has exposed a component, it can be used by any number of app plugins without any further action from the component provider.
```